### ISSUE TITLE
http://packages.tei-c.org/deb/ is not accessible

### ISSUE THREAD

**Issue by lddubeau** [TEIC/TEI#1796]
It's been a long long time since I've paid attention to the Debian packages for TEI but I recently discovered that my URLs needed updating. Checked [the wiki](https://wiki.tei-c.org/index.php/TEIDebian) but if I click on the link to get a list of packages (http://packages.tei-c.org/deb/), I get "Forbidden" as a response. So something is off. I checked the mailing list to make sure I did not miss something and noticed emails to the list about tei-c.org being down for a while. Maybe the package tree need rebuilding, or there's some configuration that is still not set for access?

**Comment by jamescummings** [comment]
Hi @lddubeau,

Yes, that is known.  I believe @peterstadler was looking at getting that changed.

**Comment by peterstadler** [comment]
Yes, sorry, that should be a temporary outage since we were releasing TEI P5 3.4.0 yesterday and in this process we were updating the deb packages, as well. Yet, we failed to recreate the Debian repo due to some missing binaries on the server. We are working on it …
